hack/release/common.sh

```diff
@@ -2,9 +2,12 @@
 set -euo pipefail
 
 config(){
+  GITHUB_ACCOUNT="aws"
   AWS_ACCOUNT_ID="071440425669"
   ECR_GALLERY_NAME="karpenter"
   RELEASE_REPO=${RELEASE_REPO:-public.ecr.aws/${ECR_GALLERY_NAME}/}
+  RELEASE_REPO_GH=${RELEASE_REPO_GH:-ghcr.io/${GITHUB_ACCOUNT}/karpenter}
```

Comment:
optional
```suggestion
  RELEASE_REPO_GH=${RELEASE_REPO_GH:-ghcr.io/aws/karpenter}
```

Reply:
Oops I missed this, but it's parametrized to make developing against a local fork easier, e.g. all you have to do is modify values in the config function.